fvm/state/state.go

```diff
@@ -0,0 +1,162 @@
+package state
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+const (
+	defaultMaxKeySize         = 16_000     // ~16KB
+	defaultMaxValueSize       = 32_000_000 // ~32MB
+	defaultMaxInteractionSize = 64_000_000 // ~64MB
```

Comment:
Maybe export these constants and import them from `fvm/context.go` above

Reply:
applied at 3265367